guides/v2.1/extension-dev-guide/routing.md

```diff
@@ -160,6 +160,21 @@ $this->_forward('action', 'controller', 'Other_Module')
 {: .bs-callout .bs-callout-tip }
 Use the [`ActionFactory`] in your router to create an instance of an `Action` class.
 
+
+{: .bs-callout .bs-callout-info }
+Action class should return a `result object`.
+
+## Result object
+
+Name | Description
+--- | ---
+`json` | Sets `Content-Type:application/json` in the header and returns a json encoded representation of an array with data.
+`raw` | Returns the data as it's been set. Does not set a `Content-Type` of the header.
```

Comment:
```suggestion
`raw` | Returns the data as it's been set. Does not set a `Content-Type` in the header.
```
Also, remove period from end of last sentence.

Reply:
**done**